nodes/src/nodes/tts_openai/IGlobal.py

```
@@ -0,0 +1,184 @@
+}
+
+
+class IGlobal(IGlobalBase):
+    """
+    Global configuration and setup for the OpenAI text-to-speech node.
+
+    Handles API client initialization, configuration validation, and
+    provides the shared OpenAI client for all instances.
+    """
+
+    def validateConfig(self):
```

Ruff (0.15.7)

[warning] Too many branches (14 > 12)

(PLR0912)